messages/context.json

```diff
@@ -22,6 +22,8 @@
   "settings.addCustomOption_Name": "Return option",
   "settings.addCustomOption_Days": "Maximum return days",
   "settings.addCustomOption_Submit": "ADD OPTION",
+  "settings.otherOptions": "Other Options",
+  "settings.otherOptions_enable": "Enable Other Options",
```

Comment:
A nitpick here, but I would phrase it as:
`'Other' option`
&
`Enable 'Other' option`
Implying that the option for 'Other' is being enabled, and not that there are 'Other options' available.

Reply:
actually i was trying to follow the same pattern of the previous settings variables, as you can see in line 24 and 23 they add underscore postfix the name of the variables, but yeah i will change it as what you see it's better.

Reply:
Sorry, not the variables itself. But the TEXT

Reply:
ohh i see, i will change it now